admin-ui/src/component/reservations/requested/hooks/index.ts

```diff
@@ -22,6 +23,29 @@ import { GQL_MAX_RESULTS_PER_QUERY } from "../../../../common/const";
 
 export { default as usePermission } from "./usePermission";
 
+const getEventName = (eventType?: string, title?: string) =>
+  eventType === "blocked" ? "Suljettu" : title?.trim();
+
+const getReservationTitle = (r: ReservationType) => r.reserveeName ?? "";
+
+const convertReservationToCalendarEvent = (r: ReservationType) => ({
+  title: getEventName(r.type ?? undefined, getReservationTitle(r)),
+  event: {
+    ...r,
+    name: r.name?.trim() !== "" ? r.name : "No name",
```

Comment:
Might want to get the string from translations. Also fi/en?

Reply:
This is fixed in another PR. it should be empty